files/__addonLocation__/package.json

```diff
@@ -25,6 +25,9 @@
     "test": "echo 'A v2 addon does not have tests, run tests in test-app'",
     "prepack": "rollup --config"
   },
+  "peerDependencies": {
+    "ember-source": "^4.12.0 || ^5.0.0"
```

Comment:
Isn't this a bit too aggressive at this point? The latest LTS is still 4.8.

Also we'd need to update [the readme](https://github.com/embroider-build/addon-blueprint/blob/main/files/README.md?plain=1#L7), the [ember-try config](https://github.com/embroider-build/addon-blueprint/blob/main/files/test-app-overrides/config/ember-try.js) and [ci.yml](https://github.com/embroider-build/addon-blueprint/blob/main/files/.github/workflows/ci.yml)

Reply:
> Also we'd need to update [the readme](https://github.com/embroider-build/addon-blueprint/blob/main/files/README.md?plain=1#L7), the [ember-try config](https://github.com/embroider-build/addon-blueprint/blob/main/files/test-app-overrides/config/ember-try.js) and [ci.yml](https://github.com/embroider-build/addon-blueprint/blob/main/files/.github/workflows/ci.yml)

good call. I'm 🤷 on what should be default here -- will update.

Reply:
Given users of the blueprint will mostly be creating new addons I think current LTS is a decent default.

Reply:
I'm widening to 3.28 || ^4.

The built in types are just for addon development, and don't cause any constraints on the peer version